### PR TITLE
Added 1d array folder in java

### LINKED ISSUE #1321: Add 1-D array folder in java

### PR DESCRIPTION
**Pull request by somya-kapoor**
Fixes #1321

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added 1-D array folder in java

#### Languages Used:
- java

#### Files Added:
- Frequency_Of_A_Element.java
- Max_And_Next_Max_Number.java 
- Mean_Variance_Standard_Deviation.java 
- Merge_Two_Sorted_Arrays.java

### CODE REVIEW

File: 1-D_Array/Java/Max_And_Next_Max_Number.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        System.out.println("Next max: " + smax);
+    } 
+    
+    public static void main(String args[]) 
+    {
+        int n;
+        Scanner s = new Scanner(System.in);
+        n = s.nextInt();
+        int arr[] = new int[n];
+        for(int i = 0; i < n; i++)
```
Comment (MastersAbh) [inline 1-D_Array/Java/Max_And_Next_Max_Number.java]: space after for

File: 1-D_Array/Java/Mean_Variance_Standard_Deviation.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        System.out.println("Standard Deviation = " + sd);
+    } 
+    
+    public static void main(String args[]) 
+    {
+        int n;
+        Scanner s = new Scanner(System.in);
+        n = s.nextInt();
+        int arr[] = new int[n];
+        for(int i = 0; i < n; i++)
```
Comment (MastersAbh) [inline 1-D_Array/Java/Mean_Variance_Standard_Deviation.java]: space after for

File: 1-D_Array/Java/Merge_Two_Sorted_Arrays.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        // copying remaining elements of array a 
+        while (i < num1) 
+            c[k++] = a[i++]; 
+
+        // copying remaining elements of array b
+        while (j < num2) 
+            c[k++] = b[j++]; 
+
+        System.out.println("New merged array: \n");
+        for (i = 0; i< (num1 + num2); i++)
```
Comment (MastersAbh) [inline 1-D_Array/Java/Merge_Two_Sorted_Arrays.java]: space before <

File: 1-D_Array/Java/Merge_Two_Sorted_Arrays.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        }
+    }
+    
+    public static void main(String args[]) 
+    {
+        int num1;
+        Scanner s = new Scanner(System.in);
+        num1 = s.nextInt();
+        int arr1[] = new int[num1];
+        for(int i = 0; i < num1; i++)
```
Comment (MastersAbh) [inline 1-D_Array/Java/Merge_Two_Sorted_Arrays.java]: space after for

File: 1-D_Array/Java/Merge_Two_Sorted_Arrays.java [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        {
+            arr1[i] = s.nextInt();
+        }
+        
+        int num2;
+        num2 = s.nextInt();
+        int arr2[] = new int[num2];
+        for(int i = 0; i < num2; i++)
```
Comment (MastersAbh) [inline 1-D_Array/Java/Merge_Two_Sorted_Arrays.java]: space after for

**Comment by somya-kapoor** [comment]
@MastersAbh Done please review.

**Pull request merged**